website/docs/reference/project-configs/profile.md

```diff
@@ -15,7 +15,7 @@ The profile your dbt project should use to connect to your <Term id="data-wareho
 * If you are developing locally: This configuration is required, unless a command-line option (i.e. `--profile`) is supplied.
 
 ## Related guides
-* [Connecting to your warehouse](dbt-cli/configure-your-profile)
+* [Connecting to your warehouse using the command line](https://docs.getdbt.com/docs/get-started/connection-profiles#connecting-to-your-warehouse-using-the-command-line)
```

Comment:
Oops, we should add that we use relative links to the style guide. Thank you and this is great otherwise!
```suggestion
* [Connecting to your warehouse using the command line](/docs/get-started/connection-profiles#connecting-to-your-warehouse-using-the-command-line)
```